invenio_app_rdm/records_ui/previewer/iiif_simple.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2022 data-futures.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Simple IIIF image preview."""
+
+from os.path import splitext
+
+from flask import current_app, render_template
+
+previewable_extensions = ['gif', 'jpg', 'jpeg', 'jp2', 'png', 'tif', 'tiff']
+
+# extensions that can be put in <img> tag - all others converted to jpg
+browser_extensions = ['gif', 'jpg', 'png']
+
+
+def can_preview(file):
+    """Check if file can be previewed by this previewer.
+
+    :param file: The file to be previewed.
+    :returns: Boolean
+    """
+    supported_extensions = ['.gif', '.jpg', '.jpeg', '.jp2', '.png',
+                            '.tif', '.tiff']
+    return file.has_extensions(*supported_extensions)
+
+
+def preview(file):
+    """Render template.
+
+    :param file: Image file to preview.
+    :returns: Rendered template with image from IIIF service.
+    """
+    ext = splitext(file.filename)[1].lower()[1:]
+    format = ext if ext in browser_extensions else 'jpg'
+    url = file.data['links']['iiif_base'] + '/full/800,/0/default.' + format
```

Comment:
I'm a bit unsure about the width hardcoding to `800`. This could also be configurable maybe? (e.g. via `IIIF_SIMPLE_PREVIEWER_WIDTH = '800'`)

Reply:
Done